lib/Styled/Button.tsx

```diff
@@ -75,7 +75,11 @@ export const StyledButton = styled.button<IStyledButtonProps>`
     color: #fff;
     background-color: ${props.theme.colorPrimary};
     border: none;
-    border-radius:20px;
+    border-radius:4px;
+        &:hover,
+    &:focus {
+
```

Comment:
Previous lines in this file put something inside `{ }`, is the intention to have an empty "body" here?